src/domains/transaction/components/IpfsDetail/IpfsDetail.tsx

```diff
@@ -32,7 +32,7 @@ export const IpfsDetail = ({ isOpen, transaction, onClose }: IpfsDetailPropertie
 				border={false}
 			/>
 
-			<TransactionFee currency={wallet.currency()} value={transaction.fee()} />
+			<TransactionFee currency={wallet.currency()} value={transaction.fee().toHuman()} />
```

Comment:
I'll make some more changes in the SDK after this PR to return this also as a human-readable value.